makefile

```diff
@@ -0,0 +1,80 @@
+.PHONY: build test push quality style unit-test integ-test
```

Comment:
Is this required? I think the Dockerfiles here should be removed and we should have only one copy of this in DLC repository. Otherwise this is a maintenance overhead.

Reply:
would keep it for now with `test` and `quality`. i think we can remove it when we have pipelines